### ISSUE TITLE
ML: Deep Model #1 (24 Layers): Re-implementing the code training code 

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#275]
The previous code ends up not converging and early stop, so it's better to implement it from base

Model Architect:

4Conv(Conv+Conv+MaxPool+Dropout)
4Dense(Dense+Dropout)
